gangplank/ocp/filer.go

```diff
@@ -258,15 +259,53 @@ func (m *minioServer) exec(ctx context.Context) error {
 	}
 
 	// Ensure the process gets terminated on shutdown
+	sigs := make(chan os.Signal, 64)
 	go func() {
-		sigs := make(chan os.Signal, 1)
 		signal.Notify(sigs, os.Interrupt, syscall.SIGTERM, syscall.SIGUSR1, syscall.SIGUSR2)
 		<-sigs
 		m.Kill()
 	}()
 
+	readyFn := func() <-chan bool {
```

Comment:
Do we need to wrap this in a function? Can we just do e.g.:

```go
readyChan := make(chan bool)
go func() {
  for {
  }
}()
...
select {
  ...
  case <- readyChan:
    return nil
}
```

Reply:
I moved it out of of the `exec` fn. 

The reason for wrapping was to confine the lexical scope of the write to the channel. The wrapping prevents any other use of the channel other than for reading from it.